src/types/table.ts

```diff
@@ -13,6 +15,16 @@ export interface TablePreferences {
   columnVisibility?: Record<string, boolean>
 }
 
+export interface TableDataEntry {
+  [key: string]: any
```

Comment:
Is `any` appropriate here? I wouldn’t think an array or other object structures would be valid?

Reply:
An array or object would probably not be rendered appropriately by default be we can still use slots to alter the way data is rendered in the cells. So yes, it can be `any`thing 😅